build.ps1

```diff
@@ -33,36 +33,31 @@ if($MSBuildFullPath -ne "")
     }
 }
 
-# msbuild path not provided, find msbuild for VS2019
+# msbuild path not provided, find msbuild for VS2022
```

Comment:
Let's update readme of the repo to include the same.

Reply:
Updated the Readme